charters/PORTABILITY/charter.md

```diff
@@ -0,0 +1,57 @@
+# Pipeline Portability Service
+
+## Description
+
+The portability service enables the confirmation that pipelines developed for the HCA DCP can be successfully ran outside of the HCA DCP.
```

Comment:
Change all instances of _portability service_ to _Portability Service_?

Editorial - what about  _The portability services confirms (or validates) that pipelines developed for the HCA DCP can successfully execute outside the HCA DCP_ - a bit more context would be helpful. I expect some people to quickly browse the **Description** for the elevator pitch. Why is this _goodness_? Where will the pipelines execute?

Reply:
Updated with more context on what principle we are trying to enable and that the system is focused on external execution and not internal testing.

using executed instead of run but thank you for the suggestion.